File: Domovoy/ClientApp/src/api/models/ApartmentRequest.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Apartment } from './Apartment';
import type { ApplicationUser } from './ApplicationUser';

export type ApartmentRequest = {
    id?: number;
    additionalContacts?: string | null;
    requester?: ApplicationUser;
    requesterId?: number;
    apartment?: Apartment;
    apartmentId?: number;
};

File: Domovoy/Models/ApartmentRequest.cs
namespace Domovoy.Models;

public class ApartmentRequest
{
    public int Id { get; set; }
    
    public string AdditionalContacts { get; set; }
    
    public ApplicationUser Requester { get; set; }
    public int RequesterId { get; set; }
    
    public Apartment Apartment { get; set; }
    public int ApartmentId { get; set; }
}

public class ApartmentRequestDTO
{
    public int Id { get; set; }
    
    public string AdditionalContacts { get; set; }
    
    public ApplicationUserViewModel Requester { get; set; }

    public ApartmentViewModel Apartment { get; set; }
}